### ISSUE TITLE
Missing error in logger.error(String, Cause[Any])

### ISSUE THREAD

**Issue by fokot** [zio/zio-logging#253]
```
.tapCause(logger.error("Error in logging slack message", _))
```
Logs just `"Error in logging slack message"`
But this logs also the error
```
.tapError(logger.throwable("Error in logging slack message", _))
```

**Comment by vigoo** [comment]
Related discussion: https://github.com/zio/zio-logging/pull/205

**Comment by jdegoes** [comment]
Fixed in 2.0.
